tests/connect/standalone-x509-error-0001.phpt

```diff
@@ -29,12 +28,13 @@ $opts = array(
         "allow_self_signed" => false,
         "cafile" => $SSL_DIR . "/ca.pem", /* Defaults to openssl.cafile */
         "capath" => $SSL_DIR, /* Defaults to openssl.capath */
-        "local_cert" => $SSL_DIR . "/src/libmongoc/tests/certificates/client.pem",
+        "local_cert" => $SSL_DIR . "/client.pem",
```

Comment:
I'm not sure why this path was incorrect before. It doesn't affect the test, since we're going to provide an invalid username anyway.